### ISSUE TITLE
🐛 Mass amount of Unresolved reference when update to v3

### ISSUE THREAD

**Issue by lninnnn** [cuvent/react-native-vision-camera#1712]
### What were you trying to do?

Updated from 2.15.4 to 3.0.0-rc.5

### Reproduceable Code

```tsx
"react-native": "0.71.8",
"react-native-worklets": "github:chrfalch/react-native-worklets#d62d76c",
//tried to use react-native-worklet-core but the package is not linked in cMake and build.gradle yet (still use the old name)
"react-native-vision-camera": "3.0.0-rc.5",
"@shopify/react-native-skia": "^0.1.202",
```

### What happened instead?

Build fail

### Build.gradle

```tsx
buildToolsVersion = "33.0.0"
        minSdkVersion = 28 // or higher
        compileSdkVersion = 33
        targetSdkVersion = 33
        androidXCore = "1.0.2"
        playServicesVersion = "17.0.0"
        RNNKotlinVersion = "1.7.20" 
        RNNKotlinStdlib = "kotlin-stdlib-jdk8"
        kotlinVersion = "1.7.20" 
        ndkVersion = "21.4.7075529"
```

### Relevant log output

```shell
Error: Command failed: ./gradlew app:installDebug -PreactNativeDevServerPort=8081
.../android/src/main/java/com/mrousavy/camera/extensions/MediaFormat.setDynamicRangeProfile.kt: (3, 40): Unresolved reference: DynamicRangeProfiles

../react-native-vision-camera/android/src/main/java/com/mrousavy/camera/extensions/MediaFormat.setDynamicRangeProfile.kt: (34, 54): Unresolved reference: TIRAMISU

react-native-vision-camera/android/src/main/java/com/mrousavy/camera/extensions/MediaFormat.setDynamicRangeProfile.kt: (35, 63): Unresolved reference: FEATURE_HdrEditing
../node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/extensions/Size+Extensions.kt: (12, 10): Unresolved reference: minBy

../react-native-vision-camera/android/src/main/java/com/mrousavy/camera/extensions/Size+Extensions.kt: (12, 22): Unresolved reference: it

../react-native-vision-camera/android/src/main/java/com/mrousavy/camera/parsers/VideoStabilizationMode.kt: (5, 48): Unresolved reference: CONTROL_VIDEO_STABILIZATION_MODE_PREVIEW_STABILIZATION

../react-native-vision-camera/android/src/main/java/com/mrousavy/camera/utils/CameraDeviceDetails.kt: (68, 54): Unresolved reference: TIRAMISU

../react-native-vision-camera/android/src/main/java/com/mrousavy/camera/utils/CameraDeviceDetails.kt: (69, 48): Unresolved reference: REQUEST_AVAILABLE_CAPABILITIES_DYNAMIC_RANGE_TEN_BIT
../react-native-vision-camera/android/src/main/java/com/mrousavy/camera/utils/CameraDeviceDetails.kt: (70, 49): Not enough information to infer type variable T
```

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':react-native-vision-camera:compileDebugKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

### Device

Android

### VisionCamera Version

3.0.0-rc05

### Additional information

- [ ] I am using Expo
- [X] I have read the [Troubleshooting Guide](https://react-native-vision-camera.com/docs/guides/troubleshooting)
- [X] I agree to follow this project's [Code of Conduct](https://github.com/mrousavy/react-native-vision-camera/blob/main/CODE_OF_CONDUCT.md)
- [X] I searched for [similar issues in this repository](https://github.com/mrousavy/react-native-vision-camera/issues) and found none.

**Comment by BharatDawai** [comment]
and progress on this ?

**Comment by ObidosDev** [comment]
Same for me
```
node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/extensions/Size+Extensions.kt: (12, 10): Unresolved reference: minBy
```

**Comment by ObidosDev** [comment]
To fix this need to update `kotlinVersion = '1.7.20'`

**Comment by t4dek** [comment]
@ObidosDev where exactly should we set kotlinVersion = '1.7.20' ?

Because I see the code inside library that is doing it already if I am not mistaken.

`def kotlin_version = rootProject.ext.has('kotlinVersion') ? rootProject.ext.get('kotlinVersion') : project.properties['VisionCamera_kotlinVersion']`

```
# When configured, Gradle will run in incubating parallel mode.
# This option should only be used with decoupled projects. More details, visit
# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
# org.gradle.parallel=true
#Fri Feb 19 20:46:14 CET 2021
VisionCamera_kotlinVersion=1.7.20
android.enableJetifier=true
android.useAndroidX=true
```

**Comment by lninnnn** [comment]
> To fix this need to update `kotlinVersion = '1.7.20'`

my build gradle (check above) is already declared kotlinVerson = 1.7.20

**Comment by lninnnn** [comment]
@mrousavy Hi! Do you have time to look into this?

**Comment by mrousavy** [comment]
Hi! No, why?

**Comment by keisan1231** [comment]
`node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/extensions/Size+Extensions.kt: (12, 10):`

Unresolved reference: minBy

Android Studio. This error😅


` "react-native-vision-camera": "^3.0.0",`

**Comment by mrousavy** [comment]
Closing as this is a stale issue - this might have been fixed with the full rewrite in VisionCamera V3 (🥳) - if not, please create a new issue.

If your issue has been fixed, consider [sponsoring me on GitHub](https://github.com/sponsors/mrousavy) to say thanks 💖

**Comment by interstates21** [comment]
It's not fixed in 3.6.4

Upd: 
ext.kotlinVersion = '1.7.20' in build.gradle helped!

I was overriding this kotlinVersion somewhere else 🥲
